packages/cli/commands/executeBatch.ts

```diff
@@ -659,7 +652,7 @@ export class ExecuteBatch extends Command {
 		});
 
 		return new Promise(async (resolve) => {
-			if (startNode === undefined) {
+			if (startingNode === undefined) {
```

Comment:
We dont really need this check since we throw an error if it doesn't exist. Maybe we should wrap that call in a try...catch block?

Reply:
Thanks, removed for now.